oemoflex/model_structure/links.csv

```diff
@@ -0,0 +1,22 @@
+link
+AT-CH
+AT-CZ
+AT-IT
+BE-FR
+BE-LU
+BE-NL
+CH-FR
+CH-IT
+CZ-PL
+DE-AT
+DE-BE
+DE-CH
+DE-CZ
+DE-DK
+DE-FR
+DE-LU
+DE-NL
+DE-PL
+DK-NL
+FR-IT
+FR-LU
```

Comment:
We should use this as we started the implementation

Reply:
Used now.